.github/workflows/tests.yml

```diff
@@ -33,6 +33,12 @@ jobs:
       - name: Build
         run: yarn build
 
+      - name: Generate compression codebooks
+        id: codebooks
+        working-directory: ./packages/adblocker/
+        run: |
+          yarn generate-codebooks
+
```

Comment:
This takes a very long time so probably not something we want to do on every CI flow.